core/src/main/scala/stryker4s/config/Config.scala

```diff
@@ -21,7 +21,9 @@ final case class Config(
     legacyTestRunner: Boolean = false,
     scalaDialect: Dialect = dialects.Scala3,
     concurrency: Int = Config.defaultConcurrency,
-    debug: DebugOptions = DebugOptions()
+    debug: DebugOptions = DebugOptions(),
+    staticTmpDir: Boolean = false,
```

Comment:
Could you also add a test for the new options in `ConfigReaderTest` and add docs for them in `docs/configuration.md`? Then I think we can merge this!

Reply:
Done.